kql/__init__.py

```diff
@@ -14,31 +15,49 @@
 __version__ = '0.1.4'
 __all__ = (
     "ast",
-    "to_eql",
-    "lint",
-    "parse",
     "from_eql",
     "get_evaluator",
     "KqlParseError",
     "KqlCompileError",
+    "lint",
+    "parse",
+    "to_dsl",
+    "to_eql",
 )
 
 
+def to_dsl(parsed, optimize=True, schema=None):
+    """Convert KQL to Elasticsearch Query DSL."""
+    if not isinstance(parsed, ast.KqlNode):
+        parsed = parse(parsed, optimize, schema)
+
+    return ToDsl.convert(parsed)
```

Comment:
```suggestion
    return {"query": ToDsl.convert(parsed)}
```

Reply:
> Nothing is hooked up to the existing APIs. To use it, you need to nest the output underneath a query: {...} when building a request. Or put it in a different context when doing aggregations, etc.

I wanted to convert just the expression, but not make assumptions about the context. Then you can embed it within aggregations or do whatever you want. I think down the road we could add another command to embed the expression in a full DSL